doc/Comments.xml

```diff
@@ -116,9 +116,9 @@ As an example, a full &AutoDoc; comment with all options could look like this:
 <Listing><![CDATA[
 #! @Description
 #! Computes the list of lists of degrees of ordinary characters
-#! associated to the <A>p</A>-blocks of the group <A>G</A>
-#! with <A>p</A>-modular character table <A>modtbl</A>
-#! and underlying ordinary character table <A>ordtbl</A>.
+#! associated to the <M>p</M>-blocks of the group <M>G</M>
```

Comment:
```suggestion
#! associated to the $p$-blocks of the group $G$
```
I am not sure if this is reasonable or should not be shown here.

Reply:
Not sure what you mean with that... ? Are you wondering whether to use `$` here or not? Well, my personal stance is that I like your suggestion, after all, we want to show people how "true" AutoDoc comments look like. So, once we implement single backtick support with automagic distinction between arguments, keywords, code, I'd also not hesitate to use those here instead of `<A>` and `<C>`

Reply:
Yep, you are right. I was just not sure if we should use markdown in one of the first examples, but you are right, we should directly show people how to use AutoDoc